shaders/src/surface_getters.vs

```diff
@@ -24,7 +24,7 @@ float getObjectUserData() {
 #if __VERSION__ >= 300
 /** @public-api */
 int getVertexIndex() {
-#if defined(TARGET_METAL_ENVIRONMENT) || defined(TARGET_VULKAN_ENVIRONMENT)
+#if defined(TARGET_METAL_ENVIRONMENT) || defined(TARGET_VULKAN_ENVIRONMENT) || defined(TARGET_WEBGPU_ENVIRONMENT)
```

Comment:
recommend not adding it because I don't know if this is supported in wgsl, and which would actually be the correct call.